ingest-app/docs/api.md

```diff
@@ -1225,17 +1226,20 @@ curl -XPUT \
 ```
 
 ### <a name="publish-variable-draft"></a> Publish Variable Draft 
-#### <a name="publish-variable-draft-endpoint"></a> /collections/&lt;collection-concept-id&gt;/&lt;collection-revision-id&gt;/variables/&lt;native-id&gt;/publish/&lt;variable-draft-concept-id&gt; 
+#### <a name="publish-variable-draft-endpoint"></a> /publish/&lt;variable-draft-concept-id&gt;/&lt;native-id&gt;
```

Comment:
It might be more clear to only have a singular endpoint called out in the docs. There are no longer a "publish non-variable draft endpoint" and a "publish variable draft endpoint", there is only a "publish draft endpoint". 

I think you just need a sub-section within that for "When publishing variable drafts", where you can explain the addition requirements for the body

Reply:
Good point, Thanks. Done.